pptx/text/text.py

```diff
@@ -582,7 +589,10 @@ def indent(self):
         margin of the paragraph.  If assigned |None|, the default value is
         used, 0.  Negative values can also be used.
         """
-        return self._pPr.indent
+        indent = self._pPr.indent
+        if indent is None:
+            return 0
```

Comment:
I'm curious why you chose to return 0 instead of None? Is this how Steve is doing it?

Reply:
So effectively, yes that was how he was doing it, just not implemented like this.  After some more digging, I found there is a default value you can set for optional parameters in the `oxml` level of the code.  Implemented properly now and tests are passing.